dotCMS/src/main/java/com/dotmarketing/tag/ajax/TagAjax.java

```diff
@@ -414,7 +415,13 @@ public static void deleteTagFullCommand(String tagName) throws DotDataException,
 		    	if(!tagName.toLowerCase().contains("tag name") && !hostId.toLowerCase().contains("host id")){
 		    		tagName = tagName.replaceAll("\'|\"", "");
 		    		hostId = hostId.replaceAll("\'|\"", "");
-					APILocator.getTagAPI().getTagAndCreate(tagName, "", hostId);
+		    		try {
+					    APILocator.getTagAPI().getTagAndCreate(tagName, "", hostId);
+					}catch(GenericTagException e){

```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)